initrd/dockerfile.go

```diff
@@ -504,6 +504,11 @@ func (initrd *dockerfile) Build(ctx context.Context) (string, error) {
 	)
 	initrd.env = img.Metadata.Config.Config.Env
 
+	// Remove the shell command if it is the first argument
+	if len(initrd.args) > 2 && initrd.args[0] == "/bin/sh" && initrd.args[1] == "-c" {
+		initrd.args = initrd.args[2:]
+	}
+
```

Comment:
What if the user has explicitly set this?

Reply:
good question

Reply:
Following offline conversations:
1. Can you please add a TODO note to indicate that this should be removed once shell scripts are supported; and,
2. Switch `>` to `>=`